public/js/pimcore/object/classes/data/multiselect.js

```diff
@@ -256,84 +394,128 @@ pimcore.object.classes.data.multiselect = Class.create(pimcore.object.classes.da
+                    width: 40,
+                    items: [
+                        {
+                            tooltip: t('up'),
+                            icon: "/bundles/pimcoreadmin/img/flat-color-icons/up.svg",
+                            handler: function (grid, rowIndex) {
+                                if (rowIndex > 0) {
+                                    var rec = grid.getStore().getAt(rowIndex);
+                                    grid.getStore().removeAt(rowIndex);
+                                    grid.getStore().insert(--rowIndex, [rec]);
+                                    model.select(rowIndex);
+                                }
+                            }.bind(this)
+                        }
+                    ]
+                },
+                {
+                    xtype: 'actioncolumn',
+                    menuText: t('down'),
+                    width: 40,
+                    items: [
+                        {
+                            tooltip: t('down'),
+                            icon: "/bundles/pimcoreadmin/img/flat-color-icons/down.svg",
+                            handler: function (grid, rowIndex) {
+                                if (rowIndex < (grid.getStore().getCount() - 1)) {
+                                    const rec = grid.getStore().getAt(rowIndex);
+                                    grid.getStore().removeAt(rowIndex);
+                                    grid.getStore().insert(++rowIndex, [rec]);
+                                    this.defaultValueModel.select(rowIndex);
+                                }
+                            }.bind(this)
+                        }
+                    ]
+                },
+                {
+                    xtype: 'actioncolumn',
+                    menuText: t('remove'),
+                    width: 40,
+                    items: [
+                        {
+                            tooltip: t('remove'),
+                            icon: "/bundles/pimcoreadmin/img/flat-color-icons/delete.svg",
+                            handler: function (grid, rowIndex) {
+                                grid.getStore().removeAt(rowIndex);
+                            }.bind(this)
+                        }
+                    ]
+                }
+            ],
+            autoHeight: true,
+            plugins: [
+                Ext.create('Ext.grid.plugin.CellEditing', {
+                    clicksToEdit: 1,
+                    listeners: {
+                        edit: function(editor, e) {
+                            if(!e.record.get('value')) {
+                                e.record.set('value');
+                            }
+                        },
+                    }
+                })
+            ]
+        });
     },
-
-    showoptioneditor: function (valueStore) {
-        var editor = new pimcore.object.helpers.optionEditor(valueStore);
-        editor.edit();
-    }
```

Comment:
need to revert this and rename to  `showgrideditor` from `showoptioneditor`, otherwise i get 
![image](https://github.com/user-attachments/assets/7315aa8d-d7ad-424d-b9ff-e3ca6ec34736)
from https://github.com/pimcore/admin-ui-classic-bundle/pull/252/files#diff-3cabd22d0fd1dda6a0151f733e10f8b4472af24bb72e5c8b8d67dc55a6284335R433-R436

as [pointed out](https://github.com/pimcore/admin-ui-classic-bundle/pull/252#issuecomment-2337428052) by my former coworker

Reply:
or keep it deleted and remove 
https://github.com/pimcore/admin-ui-classic-bundle/pull/252/files#diff-3cabd22d0fd1dda6a0151f733e10f8b4472af24bb72e5c8b8d67dc55a6284335R433-R435
it we don't want to see the comma separated list with `,something` like the other comment